pkg/webhook/genericshared.go

```diff
@@ -0,0 +1,27 @@
+package webhook
```

Comment:
Since it's the same package, does this require a separate file?

Reply:
this func is used by both SimpleEvent and CloudEvent, what would be the `idiomatic` :) way to have it?

Reply:
Since it's the same package, and it's not such large functionality, I think you can leave it in one of the two files (ideally the first one that implemented it).